demos/CMakeLists.txt

```diff
@@ -18,9 +18,9 @@
 #=============================================================================
 
 install(FILES
-      All_Dudes.mscz
+      Fugue_1.mscx
       Reunion.mscz
-      Triumph.mscz
+      Unclaimed_Gift.mscx
```

Comment:
But you're leaving All_Dudes.mscz and Triumph.mscz in the repository?

Reply:
If they are already in the repository then it is too late... unless they need to be changed of course, in which case they should be replaced with MSCX.

Reply:
I hate removing history anyhow.  There are other obsolete files there too.